README.md

```diff
@@ -600,7 +602,7 @@ If your project uses [SQLModel], then `Session` dependency is an SQLModel sessio
 # Pytest fixtures
 
 This library provides a set of utility fixtures, through its PyTest plugin, which is
-automatically installed with the library.
+automatically installed with the library. Using the plugin requires the `pytest_plugin` extra.
```

Comment:
I guess we don't consider this a breaking change because the extra is there, and most people would actually have this installed directly instead of via this lib?

Reply:
It's already the case that this is required as I found out in [this PR](https://github.com/dialoguemd/guestlist/pull/298). Removing `alembic` from a repo using the pytest plugin fails. So offering this is not a breaking change but documenting the current state.